src/EnergyPlus/Material.hh

```diff
@@ -343,6 +388,221 @@ namespace Material {
         // material
         int GlassSpecAngBRefleDataPtr = 0; // Data set index of back reflectance as a function of spectral and angle associated with a window glass
         // material
+        virtual bool dummy()
+        {
+            return true;
+        }
```

Comment:
dummy?

Reply:
I need a virtual function to do dynamic_cast